src/core/Akka.TestKit/TestKitBase_AwaitAssert.cs

```diff
@@ -109,7 +78,7 @@ public async Task AwaitAssertAsync(Action assertion, TimeSpan? duration=null, Ti
         /// <param name="assertion">The action.</param>
         /// <param name="duration">The timeout.</param>
         /// <param name="interval">The interval to wait between executing the assertion.</param>
-        public async Task AwaitAssertAsync(Func<Task> assertion, TimeSpan? duration=null, TimeSpan? interval=null)
+        public async ValueTask AwaitAssertAsync(Func<Task> assertion, TimeSpan? duration=null, TimeSpan? interval=null)
```

Comment:
Can't change the return value of existing methods, need to add CancellationToken support.